### DISCUSSION TITLE
Linux VPN Azure

### DISCUSSION THREAD

**Discussion by delphianaus** [SoftEtherVPN/SoftEtherVPN#1620] [General]
In the windows version there is a way to register for VPN Azure how do we do that so hostname.vpnazure.net connects to a vpn setup in linux ?

**Comment by davidebeatrici** [comment]
It should work just like on Linux.

**Reply by delphianaus** [reply]
I have it working last night , using VPNCMD. mind you very limited documentation on how to set things up would be great if there was some gui for linux for this.

**Reply by davidebeatrici** [reply]
Ideally we should implement everything in the web GUI, but in the meantime you can connect to the server using the GUI program through Wine or on Windows.

**Reply by chipitsine** [reply]
I use GUI from windows to manage linux vpn boxes

**Comment by delphianaus** [comment]
I doubt its going to be a good experience running it on pi 4b x64 Rasberian
using Wine etc. i got it working now.
